ape_safe/client/mock.py

```diff
@@ -116,3 +132,41 @@ def estimate_gas_cost(
         self, receiver: AddressType, value: int, data: bytes, operation: int = 0
     ) -> Optional[int]:
         return None  # Estimate gas normally
+
+    def get_delegates(self) -> Dict[AddressType, List[AddressType]]:
+        return self.delegates
+
+    def delegator_for_delegate(self, delegate: AddressType) -> Optional[AddressType]:
+        for delegator, delegates in self.delegates.items():
+            if delegate in delegates:
+                return delegator
+
+        return None
+
+    def add_delegate(self, delegate: AddressType, label: str, delegator: AccountAPI):
+        delegate = self.conversion_manager.convert(delegate, AddressType)
+
+        if delegator.address not in self.safe_details.owners:
+            raise SafeClientException(f"'{delegator}' not a valid owner.")
+
+        if delegator.address in self.delegates:
+            self.delegates[delegator.address].append(delegate)
+
+        else:
+            self.delegates[delegator.address] = [delegate]
+
+    def remove_delegate(self, delegate: AddressType, delegator: AccountAPI):
+        delegate = self.conversion_manager.convert(delegate, AddressType)
```

Comment:
this line suggests other types are allowed here, maybe is hiding a the true type in the method signature

Reply:
88795ef